plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.test.ts

```diff
@@ -625,6 +625,7 @@ describe('publish:github', () => {
       logger: mockContext.logger,
       defaultBranch: 'master',
       requireCodeOwnerReviews: false,
+      requiredStatusChecks: [],
```

Comment:
Should be able to remove these with the `?` on the input type.

Reply:
I wasn't able to remove these, since `enableBranchProtectionOnDefaultRepoBranch` is [always called with a value](https://github.com/backstage/backstage/blob/fc82237898ba21ad1e94ca2079a72fd41b99e441/plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts#L331).